framework/decode/vulkan_replay_options.h

```diff
@@ -463,17 +454,19 @@ struct VulkanReplayOptions : public ReplayOptions
     uint32_t                     screenshot_width, screenshot_height;
     float                        screenshot_scale;
     std::string                  replace_dir;
+    std::string                  dump_resources;
+
+    std::vector<uint64_t>                           BeginCommandBuffer_Indices;
+    std::vector<std::vector<uint64_t>>              CmdDraw_Indices;
+    std::vector<std::vector<std::vector<uint64_t>>> RenderPass_Indices;
+    std::vector<std::vector<std::vector<uint64_t>>> NextSubPass_Indices;
```

Comment:
Please see comment in relevant internal document regarding render pass related indices